### DISCUSSION TITLE
How to close Bootstrap 5 Modal by using js code

### DISCUSSION THREAD

**Discussion by jt4000** [twbs/bootstrap#35724] [General]
Goal:
Close the bootstrap 5 modal with javascript code after displaying the alert message.

Problem:
I cannot make the function myFunction to be working and also closing the modal after displaying alert message with javascript code.

Info:
*You should not use 'data-bs-dismiss="modal"'

JSBIN:
https://jsbin.com/sileqajiho/edit?html,js,output

Thank you!

-----------------
<!DOCTYPE html>
<html lang="en">
<head>
  <title>Bootstrap Example</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet">
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"></script>
</head>
<body>

<div class="container mt-3">
  <h3>Modal Example</h3>
  <p>Click on the button to open the modal.</p>
  
  <button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#myModal">
    Open modal
  </button>
</div>

<div class="modal" id="myModal">
  <div class="modal-dialog">
    <div class="modal-content">

      
      <div class="modal-header">
        <h4 class="modal-title">Modal Heading</h4>
        <button type="button" class="btn-close" data-bs-dismiss="modal"></button>
      </div>

      
      <div class="modal-body">
        Modal body..
      </div>

      
      <div class="modal-footer">
        <button type="button" class="btn btn-danger" onclick="myFunction()" >Start</button>
      </div>

    </div>
  </div>
</div>

<script>

  function myFunction() {
    alert("test")
  }
  
<script>

</body>
</html>

---------------

  
var myModal = document.getElementById('staticBackdrop');
var modal = bootstrap.Modal.getOrCreateInstance(myModal)
modal.show()

**Comment by RakeshPotnuru** [comment]
Missing closing `script` tag
```js
<script>

  function myFunction() {
    alert("test")
  }
  
</script> // missing here
```
